src/helpers/parser.ts

```diff
@@ -29,8 +29,26 @@ export const gitLinkedIssueParser = async ({ owner, repo, pull_number }: GitPars
       return null;
     }
 
-    const issueUrl = linkedIssues[0].querySelector("a")?.attrs?.href || "";
-    return issueUrl;
+    for (const linkedIssue of linkedIssues) {
+      const issueUrl = linkedIssue.querySelector("a")?.attrs?.href;
+
+      if (!issueUrl) continue;
+
+      const parts = issueUrl.split("/");
+
+      // check if array size is at least 4
+      if (parts.length < 4) continue;
+
+      // extract the organization name and repo name from the link:(e.g. "
+      const issueOrganization = parts[parts.length - 4];
+      const issueRepository = parts[parts.length - 3];
+
+      if (`${issueOrganization}/${issueRepository}` !== `${owner}/${repo}`) continue;
+
+      return issueUrl;
```

Comment:
maybe it would be better the function returns all linked issues instead of just one?

Reply:
So.. one pr has several linked issues.

In this case, should we assign the pr opener to that all linked issues?

@pavlovcik , @whilefoo

Reply:
Agree with @whilefoo's comment. As a helper function, that would be better to return all the linked issues instead of the first-come issue. 

> In this case, should we assign the pr opener to that all linked issues?

If the linked issues are bounties, I would say yes but I think we don't have such case yet. so as of now, no need to worry about that. Just update the return type string[] and return all the linked issue urls.

Reply:
The reason why I'm worrying about it is other function(automatically assigning pr author to issue) is using this issueParser.

Reply:
I'm thinking about a scenario where a PR resolves two issues so I guess it would make sense to assign the PR author to both linked issues.

Reply:
We need to think about some restrictions of max assign issue counts for the individual hunter

Reply:
Is it possible to check how many X assigns does a hunter have available and only assign them to X number of linked issues

Reply:
But we need to think how to prioritize them. which issues to assign.

Reply:
It probably doesn't matter that much, if it's an issue we can fix it later